integration/client/client.go

```diff
@@ -177,6 +169,54 @@ func (c *Client) pushLogLine(line string, timestamp time.Time, structuredMetadat
 	return fmt.Errorf("request failed with status code %v: %w", res.StatusCode, errors.New(string(buf)))
 }
 
+// pushLogLine creates a new logline
+func (c *Client) pushOTLPLogLine(line string, timestamp time.Time, logAttributes map[string]any) error {
+	apiEndpoint := fmt.Sprintf("%s/otlp/v1/logs", c.baseURL)
+
+	logs := plog.NewLogs()
+
+	logs.ResourceLogs().AppendEmpty().Resource().Attributes().PutStr("service.name", "varlog")
+	logRecord := logs.ResourceLogs().At(0).ScopeLogs().AppendEmpty().LogRecords().AppendEmpty()
+	logRecord.SetTimestamp(pcommon.Timestamp(timestamp.UnixNano()))
+	logRecord.Body().SetStr(line)
+	if len(logAttributes) > 0 {
+		if err := logRecord.Attributes().FromRaw(logAttributes); err != nil {
+			return err
+		}
+	}
+
+	ereq := plogotlp.NewExportRequestFromLogs(logs)
+
+	data, err := ereq.MarshalJSON()
+	if err != nil {
+		return err
+	}
+	req, err := http.NewRequest("POST", apiEndpoint, bytes.NewReader(data))
+	if err != nil {
+		return err
+	}
+	req.Header.Set("Content-Type", "application/json")
+	req.Header.Set("X-Scope-OrgID", c.instanceID)
+
+	// Execute HTTP request
+	res, err := c.httpClient.Do(req)
+	if err != nil {
+		return err
+	}
+
+	if res.StatusCode/100 == 2 {
+		defer res.Body.Close()
+		return nil
+	}
+
+	buf, err := io.ReadAll(res.Body)
+	if err != nil {
+		return fmt.Errorf("reading request failed with status code %v: %w", res.StatusCode, err)
+	}
+
+	return fmt.Errorf("request failed with status code %v: %w", res.StatusCode, errors.New(string(buf)))
```

Comment:
what is the difference between these two formats? I believe we can just skip creating error from the string and use the string directly in the error message
```suggestion
	return fmt.Errorf("request failed with status code %v: %s", res.StatusCode, string(buf))
```

Reply:
`%w` is used for wrapping the errors, which the consumer can then unwrap. I just copied the code from the other push function. Wrapping the error here doesn't make sense, so I will change it.